scripts/ps.py

```diff
@@ -85,7 +87,7 @@ def main():
         status = p.info['status'][:5] if p.info['status'] else ''
 
         line = templ % (
-            user[:10],
+            user,
```

Comment:
This is needed in order to keep the formatting sane in case the username is too long. Other than this, LGTM.

Reply:
We already limit `user` to 9 characters in line 75 so this is redundant.

Reply:
OK